deployments/launcher/pom.xml

```diff
@@ -263,6 +263,15 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <configuration>
+          <rules>
+	  </rules>
+          <skip>true</skip>
```

Comment:
suggests to use property to control this skip value.

Reply:
This will always fail if I open it, it says: can't resolve org.commonjava.indy.rest:indy-rest-api:yaml:1.9.9, org.commonjava.indy.rest:indy-rest-api:json:1.9.9, I guess those artifact is only needed when running indy.

Reply:
Are there times when we would NOT want to skip this, though? In other cases, we'd like to avoid releasing with dependencies that are snapshots, for instance.

Maybe we should try to address why the enforcer plugin invocation is failing, so we don't accidentally open the door to building a binary that we can't reproduce (by including snapshots from outside the codebase).